src/main/resources/config/liquibase/consolidate-changelogs.sh

```
@@ -0,0 +1,185 @@
+    -p $MYSQL_DEV_PORT:3306 mysql:9 \
+    --lower_case_table_names=1 --character_set_server=utf8mb4 --collation-server=utf8mb4_unicode_ci --explicit_defaults_for_timestamp
+docker run -d --name av-mysql-new -e MYSQL_ALLOW_EMPTY_PASSWORD=yes -e MYSQL_DATABASE=Artemis \
+    -p $MYSQL_NEW_PORT:3306 mysql:9 \
+    --lower_case_table_names=1 --character_set_server=utf8mb4 --collation-server=utf8mb4_unicode_ci --explicit_defaults_for_timestamp
+docker run -d --name av-pg-dev -e POSTGRES_USER=artemis -e POSTGRES_PASSWORD=artemis -e POSTGRES_DB=Artemis \
+    -p $PG_DEV_PORT:5432 postgres:18
+docker run -d --name av-pg-new -e POSTGRES_USER=artemis -e POSTGRES_PASSWORD=artemis -e POSTGRES_DB=Artemis \
+    -p $PG_NEW_PORT:5432 postgres:18
+
+echo "  Waiting for databases..."
+for i in $(seq 1 60); do
```

Shellcheck (0.11.0)

[warning] i appears unused. Verify use (or export if used externally).

(SC2034)